plugins/generator-1.19.2/forge-1.19.2/features/block_predicate_replaceable.json.ftl

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "minecraft:replaceable"<#if (field$x != "0")||(field$y != "0")||(field$z != "0")>,

```

Comment:
If should probably be in a new line for better code readability, formatted will put the comma in the right place anyways, applies to all cases like this one